powder_reduction.py

```diff
@@ -386,17 +403,17 @@ def process_vanadium_data(vanadium, empty_instr,
     groupvana = gvana.min('detector')
 
     assert groupvana == gvana.max('detector'), \
-        xs"Calibration table has mismatching group for detectors in same spectrum"
+        "Calibration table has mismatching group for detectors in same spectrum"
 
-    vana_rebin.coords['group'] = groupvana.data
-    vana_rebin.masks['mask'] = maskvana.data
+    vana_dspacing.coords['group'] = groupvana.data
+    vana_dspacing.masks['mask'] = maskvana.data
 
-    # Mask negative measured values
-    vana_rebin.masks['negative_bins'] = sc.Variable(dims=['spectrum', 'd-spacing'],
-                                                    values=np.less(vana_rebin.values, 0.0))
+    # # Mask negative measured values
+    # vana_rebin.masks['negative_bins'] = sc.Variable(dims=['spectrum', 'd-spacing'],
+    #                                                 values=np.less(vana_rebin.values, 0.0))
 
     # Focus
-    focused_vana = sc.groupby(vana_rebin, group='group').flatten('spectrum')
+    focused_vana = sc.groupby(vana_dspacing.unaligned, group='group').flatten('spectrum')
```

Comment:
See previous comment, should be able to use `groupby(..).sum(..)` instead of going back to unaligned data.